net-misc/frr/frr-8.0.ebuild

```diff
@@ -33,16 +33,19 @@ COMMON_DEPEND="
 	rpki? ( >=net-libs/rtrlib-0.6.3[ssh] )
 	snmp? ( net-analyzer/net-snmp:= )
 "
+
```

Comment:
Let's not add these empty lines, they unnecessarily increase vertical whitespace